File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/typescript/ADV-13.js
"use strict";
class A {
    v;
    constructor(v) {
        this.v = v;
    }
}
class B {
    v;
    constructor(v) {
        this.v = v;
    }
}
console.log("ADV-START");
const a = new A(42n);
const u = a;
const b = u;
console.log("OBS=V:" + b.v);
console.log("ADV-END");

File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/rust/ADV-13.rs
use std::io::Write;

struct A {
    v: i64,
}

struct B {
    v: i64,
}

fn main() {
    let mut out = std::io::stdout();
    writeln!(out, "ADV-START").unwrap();
    out.flush().unwrap();

    let a = A { v: 42 };
    let r: Box<dyn std::any::Any> = Box::new(a);
    let b: Box<B> = r.downcast::<B>().unwrap();
    let v: i64 = b.v;

    writeln!(out, "OBS=V:{}", v).unwrap();
    out.flush().unwrap();
    writeln!(out, "ADV-END").unwrap();
    out.flush().unwrap();
}
